mass_mailing_resend/tests/test_mass_mailing_resend.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2017-2020 Tecnativa - Pedro M. Baeza
+# Copyright 2022 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import exceptions
+from odoo.tests import common
+
+
+class TestMassMailingResend(common.TransactionCase):
```

Comment:
Please inherit from [BaseCommon](https://github.com/odoo/odoo/blob/9748b45b24140416fc6e0e87c25c60285bef3d70/odoo/addons/base/tests/common.py#L20C7-L20C17)
```suggestion
class TestMassMailingResend(BaseCommon):
```